src/constants/coin-config.js

```diff
@@ -102,6 +102,7 @@ export default {
   },
   supportedRecoveries: {
     crossChain: ['btc', 'bch', 'ltc'],
-    nonBitGo: ['btc', 'eth', 'xrp', 'bch', 'xlm', 'ltc', 'dash', 'zec', 'btg', 'token']
+    nonBitGo: ['btc', 'eth', 'xrp', 'bch', 'xlm', 'ltc', 'dash', 'zec', 'btg', 'token'],
+    unsignedSweep: ['btc', 'bch', 'ltc', 'dash', 'zec', 'btg', 'eth', 'token']
```

Comment:
XRP? XLM?

Reply:
the customer isn't supporting XRP and XLM at first

Reply:
This is still a useful feature for other customers though. We have other customers who use self-custody and would like a utility to recover their coin in case of emergency.

Reply:
i think i'd rather include the xrp and xlm on a later release (perhaps with the upcoming ledger fix). a separate customer has been blocked for like 2 weeks because they're waiting on us to release a version of WRW that recovers segwit unspents, so i'd rather not delay this release any more